src/components/homepage/homepage-body.jsx

```diff
@@ -0,0 +1,236 @@
+import React from 'react';
+
+const HomepageBody = () => {
+	return (
+		<main className='homepage-content'>
+			<section>
+				<div className='card-overlay-holder image'>
+					<img src='https://gmedia.playstation.com/is/image/SIEPDC/ps5-games-hero-banner-desktop-03-en-07oct21?$2400px$' alt='' srcSet='' />
+					<div className='card-overlay'>
+						<h1>NEED GAMES IS ALL YOU NEED!</h1>
+						<a href='#feature-section' className='btn btn-link flex-column text-align-center'>
+							Checkout Featured
+						</a>
+					</div>
+				</div>
+			</section>
+			<a className='btn btn-primary flex-column text-align-center' href='./screens/product-listing-page/product-listing-page.html'>
+				<h3> Find More Products-{'>'} </h3>
+			</a>
+			<h2>Featured Items</h2>
+
+			<section id='feature-section' className='feature-section'>
+				<div className='features-row'>
+					<div className='card card-md'>
+						<div className='card-body card-overlay-holder'>
+							<img className='card-img' src='/assets/images/games/HFW-1.png' loading='lazy' alt='video game image1' />
+							<div className='card-overlay'>
+								<p>Releasing:TBH</p>
+								<p>Available: PS5 & PS4</p>
+							</div>
+						</div>
+
+						<div className='card-badge'>
+							<p>Pre Order Now!</p>
+						</div>
+					</div>
+
+					<div className='card card-md'>
+						<div className='card-body card-overlay-holder'>
```

Comment:
<div className='card card-md'>

lot of repetition of this div. can be converted to a component

Reply:
Yep, will be making them into one component. Thanks for the suggestion,